_config.yml

```diff
@@ -1,15 +1,13 @@
-title: Your awesome title
-author: GitHub User
-email: your-email@domain.com
+title: This is Malind's Site
+author: M Kowalski  
+email: malind.nikulski@gmail.com
 description: > # this means to ignore newlines until "baseurl:"
-  Write an awesome description for your new site here. You can edit this
-  line in _config.yml. It will appear in your document head meta (for
-  Google search results) and in your feed.xml site description.
+  Welcome to this! I'm learning so much with Github.
```

Comment:
Also, since this line is moved up, make sure to delete it here.

Reply:
@brianamarie Hello again!  I feel like I must be doing something incorrectly.  I made the changes on the 2 suggestions.  I created a new pull request but I'm not seeing any actual change on the site.